lightning/Cargo.toml

```diff
@@ -13,18 +13,22 @@ Still missing tons of error-handling. See GitHub issues for suggested projects i
 [features]
 # Supports tracking channels with a non-bitcoin chain hashes. Currently enables all kinds of fun DoS attacks.
 non_bitcoin_chain_hash_routing = []
-fuzztarget = ["secp256k1/fuzztarget", "bitcoin/fuzztarget", "bitcoin_hashes/fuzztarget"]
 # Unlog messages superior at targeted level.
 max_level_off = []
 max_level_error = []
 max_level_warn = []
 max_level_info = []
 max_level_debug = []
 
+# Testing only features, don't enable these unless you want to run rust-lightning tests!
+fuzztarget = ["secp256k1/fuzztarget", "bitcoin/fuzztarget", "bitcoin_hashes/fuzztarget"]
+mutation_testing = ["mutagen"]
+
 [dependencies]
 bitcoin = "0.21"
 bitcoin_hashes = "0.7"
 secp256k1 = "0.15"
+mutagen = { git = "https://github.com/TheBlueMatt/mutagen", optional = true }
```

Comment:
I think you meant to put this in `dev-dependencies`?

Reply:
No, for some reason dev-dependencies can't be optional, and we don't want to require a nightly-only crate for any cargo test.

Reply:
right, that's one of the oldest bugs we're still stuck with somehow (cc https://github.com/rust-lang/cargo/issues/1596)